test/measures/measures-data-spec.js

```diff
@@ -31,5 +31,40 @@ describe('measures data json', function() {
 
       assert.equal(intersection.size, multiPerformanceIds.size);
     });
+
+    describe('CAHPS measures', function() {
+      const cahpsMeasures = measuresData.filter(function(measure) {
+        var re = /CAHPS_\d+/i;
+        return measure.measureId.match(re) !== null;
+      });
+      const numCahpsMeasures = cahpsMeasures.length;
+
+      function testExpectedField(fieldName, expectedFieldValue, measuresToTest, numExpected) {
+        var numExpected = numExpected || numCahpsMeasures;
+        var measuresToTest = measuresToTest || cahpsMeasures;
+        var fieldValues = measuresToTest.map((measure) => measure[fieldName]);
+        assert.deepEqual(fieldValues, Array(numExpected).fill(expectedFieldValue));
+      }
+
+      it('includes 12 cahps measures', function() {
+        assert.equal(numCahpsMeasures, 12);
+      });
+
+      it('are all floats', () => testExpectedField('metricType', 'float'));
+      it('all have measureType \'patientEngagementExperience\'', () => testExpectedField('measureType', 'patientEngagementExperience'));
+      it('all have primarySteward \'Agency for Healthcare Research & Quality\'', () => testExpectedField('primarySteward', 'Agency for Healthcare Research & Quality'));
+      it('all have submissionMethods \'certifiedSurveyVendor\'', () => testExpectedField('submissionMethods', ['certifiedSurveyVendor']));
+      it('all have measureSets \'generalPracticeFamilyMedicine\'', () => testExpectedField('measureSets', ['generalPracticeFamilyMedicine']));
+      it('all have firstPerformanceYear 2017', () => testExpectedField('firstPerformanceYear', 2017));
+      it('all have category quality', () => testExpectedField('category', 'quality'));
+      it('all have isHighPriority true', () => testExpectedField('isHighPriority', true));
+      it('all have isInverse false', () => testExpectedField('isInverse', false));
```

Comment:
would one `deepEqual` call per CAHPS measure that verifies the fields make this simpler?

```js
it('creates 12 correct CAHPS measures', function() {
  const cahpsMeasures = measuresData.filter(measure => measure.measureId.match(/CAHPS_\d+/));
  const commonCahpsProperties = {
    'measureType': 'patientEngagementExperience',
    ...
  };
  const nqfIdMap = {
    'CAHPS for MIPS SSM: Getting Timely Care, Appointments and Information': '0005',
    'CAHPS for MIPS SSM: How Well Providers Communicate': '0005',
    'CAHPS for MIPS SSM: Patient\'s Rating of Provider': '0005',
    'CAHPS for MIPS SSM: Courteous and Helpful Office Staff': '0005'
  }
  assert.equal(cahpsMeasures.length, 12);
  cahpsMeasures.forEach(cahpsMeasure => {
    assert.match(cahpsMeasure.measureTitle, /^CAHPS for MIPS/);
    // these are the same for all CAHPS measures
    assert.deepEqual(cahpsMeasure.pick(Object.keys(commonCahpsProperties)), commonCahpsProperties);
    if (nqfIdMap[cahpsMeasure.measureTitle]) {
      assert.equal(cahpsMeasure.nqfId, '0005');
    }
  });
});
```
the mocha failure message gives a nice object diff: https://mochajs.org/#diffs

also, trivia, but [passing arrow functions to mocha is discouraged](https://mochajs.org/#arrow-functions) :(